en/config/actuators.md

```diff
@@ -540,16 +563,33 @@ For each of the control surfaces:
    - After setting the trim for a control surface, move its slider away from the center, release, and then back into disarmed (middle) position.
      Confirm that surface is in the neutral position.
 
-     ```
-
-     ```
-
 ::: info
 Another way to test without using the sliders would be to set the [`COM_PREARM_MODE`](../advanced_config/parameter_reference.md#COM_PREARM_MODE) parameter to `Always`:
 
 - This will enable the control of servos even when the vehicle is disarmed, and will constantly be applying the Trim setting to the Control Surfaces
 - You can try setting different values for the Trim and check the alignment, and then settle on the value you are happy with.
-  :::
+
+:::
+
+#### Control surfaces that move from neutral to full deflection
+
+Control surfaces that move only one direction from neutral include: airbrakes, spoilers, and flaps.
+
+For these controls you should set the minimum and maximum PWM values according to the full range of the control.
+The `Disarmed` value should then match the value (of maximum or minimum) that corresponds to control being in the "neutral" position.
+For a flap, that is when the flap is fully retracted and flush with the wing.
+
+One approach for setting these up is:
+
+1. Set the `Disarmed` value so that the surfaces are at neutral position when disarmed.
+2. ...
+3. Trim?
```

Comment:
@sfuhrer I decided that the controls that deflect in only one direction from neutral are sufficiently different to warrant a new subsection. That way we can have a note for flaps and we don't have to assume the centered disarmed position.

But I can't write this, because I can't try it. I mean I have put your note above and the special note below, but I don't know the best practical steps for this?

My guess was that you might 
- set values disarmed/min/max of 1500, 1200, 1700. 
- move slider up and check it moves, and that it is extending. If not, reverse range.
- enable slider, them change `disarmed` PWM until control retracted/flush with wing. If this is closer to min the move min to same value. If close to max, them move max to same value.
- whichever max/min you didn't set to same value as disarmed represents the value when fully extended. So set slider to max, then change either max/min so that the control is fully extended when the slider is at top.

Does that make sense. There may be a safter and less confusing rule!

Reply:
This process is spot on. 

>Then what about trimming?

You don't need to trim anything anymore if you made MIN or MAX being flush in step 3)